frontend/src/upload.js

```diff
@@ -29,25 +33,48 @@ $(document).ready(function() {
     copyBtn.html('Copy');
   });
 
-  let onChange = event => {
-    const file = event.target.files[0];
-
+  // on file upload by browse or drag & drop
+  window.onUpload = event => {
+    event.preventDefault();
+    let file = '';
+    if (event.type == 'drop') {
+      file = event.dataTransfer.files[0];
```

Comment:
Spoiler alert: It doesn't.

But it looks like we can do `Array.from().forEach()` (or just a regular `for` loop) on the FileList as a workaround. But that exposes another dull edge case where you're dragging 2-3 files onto the app and it returns you a single share URL that you can copy to your clipboard.

```js
  window.onUpload = event => {
    event.preventDefault();
    let fileList = $('#uploaded-files');
    let files = [];

    if (event.type === 'drop') {
      files = event.dataTransfer.files;
    } else {
      files = event.target.files;
    }

    Array.from(files).forEach(file => { ... });
```